ArduCopter/takeoff.cpp

```diff
@@ -187,8 +187,7 @@ void Mode::auto_takeoff_run()
         }
         pos_control->relax_velocity_controller_xy();
     } else {
-        Vector2f vel;
-        Vector2f accel;
+        Vector2f vel, accel;  // defined but not used in this scope
```

Comment:
This comment is misleading.

We may not be using these methods in those scope, but their value - implicitly zero - is absolutely critical.  This is telling the position controller to set its input velocity acceleration to zero.  The fact we don't use vel and accel elsewhere in this scope is besides the point - they are *input values* to the method we are calling!